client-sdk/ts-web/rt/src/address.ts

```diff
@@ -1,18 +1,38 @@
 import * as oasis from '@oasisprotocol/client';
+import * as elliptic from 'elliptic';
+import {Keccak} from 'sha3';
 
 import * as types from './types';
 
-export const V0_SECP256K1_CONTEXT_IDENTIFIER = 'oasis-runtime-sdk/address: secp256k1';
-export const V0_SECP256K1_CONTEXT_VERSION = 0;
+export const V0_SECP256K1ETH_CONTEXT_IDENTIFIER = 'oasis-runtime-sdk/address: secp256k1eth';
+export const V0_SECP256K1ETH_CONTEXT_VERSION = 0;
 export const V0_MULTISIG_CONTEXT_IDENTIFIER = 'oasis-runtime-sdk/address: multisig';
 export const V0_MULTISIG_CONTEXT_VERSION = 0;
 
-export async function fromSecp256k1PublicKey(pk: Uint8Array) {
-    return await oasis.address.fromData(
-        V0_SECP256K1_CONTEXT_IDENTIFIER,
-        V0_SECP256K1_CONTEXT_VERSION,
-        pk,
-    );
+export const ADDRESS_PREFIX = 'oasis';
```

Comment:
imo take this from oasis.staking.ADDRESS_PREFIX the same way our go codebase does